examples/time_frequency/plot_time_frequency_erds.py

```diff
@@ -96,23 +97,42 @@ def center_cmap(cmap, vmin, vmax):
 freqs = np.arange(2, 36, 1)  # frequencies from 2-35Hz
 n_cycles = freqs  # use constant t/f resolution
 vmin, vmax = -1, 1.5  # set min and max ERDS values in plot
+baseline = [-1, 0]  # baseline interval (in s)
 cmap = center_cmap(plt.cm.RdBu, vmin, vmax)  # zero maps to white
 
 for event in event_ids:
-    power = tfr_multitaper(epochs[event], freqs=freqs, n_cycles=n_cycles,
-                           use_fft=True, return_itc=False, decim=2)
-    power.crop(tmin, tmax)
+    tfr = tfr_multitaper(epochs[event], freqs=freqs, n_cycles=n_cycles,
+                         use_fft=True, return_itc=False, average=False,
+                         decim=2)
+    tfr.crop(tmin, tmax)
+    tfr.apply_baseline(baseline, mode="percent")
 
     fig, ax = plt.subplots(1, 4, figsize=(12, 4),
                            gridspec_kw={"width_ratios": [10, 10, 10, 1]})
-    for i in range(3):
-        power.plot([i], baseline=[-1, 0], mode="percent", vmin=vmin, vmax=vmax,
-                   cmap=(cmap, False), axes=ax[i], colorbar=False, show=False)
+    average_tfr = tfr.average()
+    for i in range(3):  # for each channel
```

Comment:
you have multiple comparisons (channels) so you should correct the obtained p values too. BTW make sure that you get two-tail p values when doing separate clusterings for tails.

Reply:
@cbrnr just add a comment that it's only corrected for multiple comp within channels so @mmagnuski can shut up :D

Reply:
🤣 I'll add a comment... - is this OK with you @mmagnuski?

Reply:
sure :)

Reply:
Coming back to the question of keeping significant clusters: are you 100% sure I need to set p <= 0.025? With such a low value, I'm not getting the clusters I'd like to see, therefore I'm tempted to lower the `threshold`. Which kind of amounts to the same thing (i.e. raising p). Furthermore, the clusters I find also depend a lot on `seed`.

Reply:
just mention that you are performing six tests (channels x tails) and that you don't correct for the number of tests performed. My fear was that people will just copy-paste and increase the number of channels to 64 for example (because 3d cluster correction is not supported yet).